modules/starting-microshift-service.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+:_content-type: PROCEDURE
+[id="starting-microshift_service_{context}"]
+= Starting the MicroShift service
+
+Use the following procedure to start the MicroShift service. 
+
+.Prerequisites 
+
+* You have installed MicroShift from an RPM package. 
+
+.Procedure 
+
+. Start the MicroShift service by entering the following command: 
++
+[source,terminal]
+----
+$ sudo systemctl start microshift
+----
+
+. Optional: To configure your {op-system} machine to start MicroShift when your machine starts, enter the following command:
++
+[source,terminal]
+----
+$ sudo systemctl enable microshift
+----
+
+. Optional: To disable MicroShift from automatically starting when your machine starts, enter the following command:
++
+[source,terminal]
+----
+$ sudo systemctl disable microshift
+----
++
+[NOTE]
+====
+The first time that the MicroShift service starts, it will take a few minutes to download and initialize MicroShift's container images. On subsequent starts, the MicroShift service should only take a few seconds to become available. 
```

Comment:
I'm a little concerned by the promise of "a few seconds". Can we find a way to say "faster" without being specific as to how fast?

Reply:
Got it. This is one of those ones I knew wouldn't pass OCP peer review, but I couldn't think of a solution at the time of my revising. Fixed now.